test/js/specs/happy.js

```diff
@@ -1,16 +1,51 @@
 const path = require('path')
+const expect = require('chai').expect;
 import {describe, it} from '../utils/mochaw'
 
 const options = {
   pageObjects: ['DocumentSelection', 'Welcome', 'DocumentUpload']
 }
 
-describe('Happy Paths',options, ({driver,pageObjects}) => {
+const localhostUrl = 'https://localhost:8080/'
+
+describe('Happy Paths',options, ({driver,$,pageObjects}) => {
   const {documentSelection, welcome, documentUpload} = pageObjects
 
```

Comment:
The welcome copy can be defined here so you don't have to repeat it in each test. A part from that it looks great!
`const copy = welcome.copy()`